content/id/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -0,0 +1,684 @@
+
+   {{< codenew file="pods/pod-single-configmap-env-variable.yaml" >}}
+
+   Buat Pod:
+
+ ```shell
+ kubectl create -f https://kubernetes.io/id/examples/pods/pod-single-configmap-env-variable.yaml
+ ```
+
+   Sekarang, keluaran dari Pod meliputi variabel _environment_ `SPECIAL_LEVEL_KEY=very`.
+
+### Menentukan variabel _environment_ kontainer dengan data dari beberapa ConfigMap
+
+ * Seperti pada contoh sebelumnya, buat ConfigMap terlebih dahulu.
+
+   {{< codenew file="configmap/configmaps.yaml" >}}
+
+   Buat ConfigMap:
+
+ ```shell
+ kubectl create -f https://kubernetes.io/examples/configmap/configmaps.yaml
+ ```
+
+* Tentukan variabel _environment_ pada spesifikasi Pod.
+
+  {{< codenew file="pods/pod-multiple-configmap-env-variable.yaml" >}}
+
+  Buat Pod:
+
+ ```shell
+ kubectl create -f https://kubernetes.io/id/examples/pods/pod-multiple-configmap-env-variable.yaml
+ ```
+
+  Sekarang, keluaran Pod meliputi variabel _environment_ `SPECIAL_LEVEL_KEY=very` dan `LOG_LEVEL=INFO`.
+
+## Mengatur semua pasangan kunci-nilai pada ConfigMap sebagai variabel _environment_ kontainer
+
+{{< note >}}
+Fungsi ini tersedia pada Kubernetes v1.6 dan selanjutnya.
+{{< /note >}}
+
+* Buat ConfigMap yang berisi beberapa pasangan kunci-nilai.
+
+  {{< codenew file="configmap/configmap-multikeys.yaml" >}}
+
+  Buat ConfigMap:
+
+ ```shell
+ kubectl create -f https://kubernetes.io/examples/configmap/configmap-multikeys.yaml
+ ```
+
+* Gunakan `envFrom` untuk menentukan seluruh data pada ConfigMap sebagai variabel _environment_ kontainer. Kunci dari ConfigMap akan menjadi nama variabel _environment_ di dalam Pod.
+
+ {{< codenew file="pods/pod-configmap-envFrom.yaml" >}}
+
+ Buat Pod:
+
+ ```shell
+ kubectl create -f https://kubernetes.io/examples/pods/pod-configmap-envFrom.yaml
+ ```
+
+ Sekarang, Pod keluaran pod meliputi variabel _environment_ `SPECIAL_LEVEL=very` dan `SPECIAL_TYPE=charm`.
+
+
+## Menggunakan variabel _environment_ yang ditentukan ConfigMap pada perintah Pod 
+
+Kamu dapat menggunakan variabel _environment_ yang ditentukan ConfigMap pada bagian `command` dari spesifikasi Pod menggunakan sintaks substitusi Kubernetes `$(VAR_NAME)`.
+
+Sebagai contoh, spesifikasi Pod berikut
+
+{{< codenew file="pods/pod-configmap-env-var-valueFrom.yaml" >}}
+
+dibuat dengan menjalankan
+
+```shell
+kubectl create -f https://kubernetes.io/examples/pods/pod-configmap-env-var-valueFrom.yaml
+```
+
+menghasilkan keluaran pada kontainer `test-container` seperti berikut:
+
+```shell
+very charm
+```
+
+## Menambahkan data ConfigMap pada Volume
+
+Seperti yang sudah dijelaskan pada [Membuat ConfigMap dari berkas](#create-configmaps-from-files), ketika kamu membuat ConfigMap menggunakan ``--from-file``, nama dari berkas tersebut akan menjadi kunci yang disimpan pada bagian `data` dari ConfigMap. Isi berkas tersebut akan menjadi nilai dari kunci tersebut.
+
+Contoh pada bagian ini merujuk pada ConfigMap bernama special-config, Seperti berikut.
```

Comment:
should we add backtick to `special-config` since it denotes part in the snippet?
```suggestion
Contoh pada bagian ini merujuk pada ConfigMap bernama `special-config`, Seperti berikut.
```

Reply:
That's right. I'll fix that.